appveyor.yml

```diff
@@ -5,6 +5,7 @@ image:
   - Visual Studio 2019
 environment:
   APPVEYOR_YML_DISABLE_PS_LINUX: true
+  CAKE_SETTINGS_SKIPPACKAGEVERSIONCHECK: true
```

Comment:
can you tell me what this is and why is it needed?

Reply:
Hi @garfield69 

If you look at any recent build, there is a warning at the start since the Jackett cake build is just using the latest version available of those packages
![image](https://user-images.githubusercontent.com/22320807/71954859-bedf0400-323a-11ea-92fa-d458477ab8a1.png)
Can remove it if you don't want it?

Reply:
oh I see. so you are saying that because of
https://github.com/Jackett/Jackett/blob/b72ad9a7f57d0d1c9149ff2c66150ba9696679d1/build.cake#L1-L2
we get the warning.
but that the nuget will always fetch the latest version since we didn't specifically ask for a particular one.
so turning off the versioncheck hides those 20 lines of log.
That's fine. Thanks.